vertx-web/src/main/java/io/vertx/ext/web/Router.java

```diff
@@ -363,10 +363,24 @@ default void accept(HttpServerRequest request) {
    *
    * @param exceptionHandler  the exception handler
    * @return a reference to this, so the API can be used fluently
+   * @deprecated you should use {@link Router#errorHandler(int, Handler)} with 500 status code
```

Comment:
do we want to remove that in Vert.x 4 ?

Reply:
yes this is the idea. Maybe we can put the new method in 3.7 and remove the old one in 4

Reply:
And of course I will reflect this changes in vertx-web-api-contract too